# Illustration of onError.
# The test function should take a string
# and parse it as an integer.
# Depending on the language, either the validation function should accept
# an error as well as an integer, or a separate error validation function
# should be provided. In either case, the function should verify
# that an error of the appropriate kind occurred.

name: errors
description: the onError field
subtests:
  - in: '17'
    want: 17

  - onError: succeed
    # onError is inherited by subtests
    subtests:
      - in: ''
      - in: 'x'

  - onError: validate
    in: 'xyz'
 
  
 
